datasets/us/fincen_special_measures/crawler.py

```diff
@@ -0,0 +1,147 @@
+from lxml import html
+import re
+from zavod import Context, helpers as h
+from normality import slugify
+from typing import Dict, Generator
+
+BASE_URL = "https://www.fincen.gov"
+
+
+def convert_date(date_str: str) -> str or None:
+    """Convert various date formats to 'YYYY-MM-DD'."""
+    # Regular expression to find dates in the format 'MM/DD/YYYY'
+    date_pattern = re.compile(r"(\d{1,2}/\d{1,2}/\d{4})")
+    match = date_pattern.search(date_str)
+    if match:
+        date_str = match.group(1)  # Extract the first matching date
+
+    formats = [
+        "%m/%d/%Y",  # 'MM/DD/YYYY' format
+        "%B %d, %Y",  # 'Month DD, YYYY' format
+        "%d-%b-%y",  # 'DD-MMM-YY' format
+    ]
+    return h.parse_date(date_str, formats)
+
+
+def crawl_item(context: Context, row: Dict[str, str]):
+    # Create the entity based on the schema
+    name = row.pop("company")
+    schema = context.lookup_value("target_type", name)
+    if schema is None:
+        schema = "Company"
+    entity = context.make(schema)
+    entity.id = context.make_id(name)
+    entity.add("name", name)
+    entity.add("topics", "sanction")
+
+    # Extract PDF links
+    pdf_link_finding = row.get("pdf_link_finding", None)
+    pdf_link_nprm = row.get("pdf_link_nprm", None)
+    pdf_link_final_rule = row.get("pdf_link_final_rule", None)
+    pdf_link_rescinded = row.get("pdf_link_rescinded", None)
+
+    # Convert relative URLs to full URLs
+    if pdf_link_finding and pdf_link_finding.startswith("/"):
+        pdf_link_finding = BASE_URL + pdf_link_finding
```

Comment:
an easier way to do this is `doc.make_links_absolute(...)`
https://github.com/opensanctions/opensanctions/blob/main/datasets/at/meine_abgeordneten/crawler.py#L185